src/edu/ucsd/library/xdre/web/CollectionOperationController.java

```diff
@@ -1072,6 +1076,10 @@ else if (validFilesCount != matchedFiles.size())
 						 }
 					 }
 					 
+					 // for single items submitting from text input, set it to higher priority .
+					 if (StringUtils.isNotBlank(txtInput))
```

Comment:
If I'm reading this right, then entering a comma-separated list of ARKs will result in normal priority reindexing, and uploading a file of ARKs or selecting collections will result in low priority indexing.

That seems fine to me -- can you add a note saying that?  So you could change "Delimiter comma ," to: "Comma-separated list of ARKs for normal priority indexing".